boards/DCM/Src/Io/Io_Imu_LSM6DS33.c

```diff
@@ -0,0 +1,189 @@
+#include "Io_Imu_LSM6DS33.h"
+#include "App_Imu.h"
+
+// The I2C handle for the I2C device the IMU is connected to
+static I2C_HandleTypeDef *imu_i2c_handle = NULL;
```

Comment:
We should includes these values inside of a struct, and then declare a static struct below. Refer to Io_LTC6813 code if you're wondering how this is done

Reply:
done